manifest/v1alpha/annotation/validation_test.go

```diff
@@ -187,16 +187,16 @@ func TestSpec_Category(t *testing.T) {
 		err := validate(annotation)
 		testutils.AssertNoError(t, annotation, err)
 	})
-	for _, category := range []string{CategoryComment, CategoryReviewNote, CategorySloEdit} {
```

Comment:
question: CategorySloEdit is added to GetSystemCategories. Is it correct? https://github.com/nobl9/nobl9-go/pull/834/changes#diff-9831e0e2d8f9cf9e450ed86813bfe658842fc3ec3bae4303f8069cc2dd0bcccaR22